### ISSUE TITLE
Per-runstep and inter-runstep comments?

### ISSUE THREAD

**Issue by jeffreykegler** [jeffreykegler/yahc#29]
Our current rule is that inter-runstep comments must be rune-aligned.  But many comments in the corpus are runstep-aligned, and it seems to make sense where the text is pertinent to the runstep which follows.  Below I have some code which illustrates per-runstep comments.

@ohAitch -- Perhaps we should allow both per-runstep and inter-runstep comments?

```
18>     :~  create+(ot nom+so des+so sec+secu ~)
19          design+(ot nom+so cof+conf ~)
20          delete+(ot nom+so why+(mu so) ~)
21          depict+(ot nom+so des+so ~)
22          filter+(ot nom+so fit+filt ~)
23          permit+(ot nom+so inv+bo sis+(as (su fed:ag)) ~)
24          source+(ot nom+so sub+bo srs+(as sorc) ~)
25          usage+(ot nom+so add+bo tas+(as so) ~)
[ hoons/arvo/mar/hall/action.hoon 27:9 Test::Whitespace tallColsig joined 0-running runstep #9 @26:9; comment overindented by 4 ]
26!         ::
27>         convey+(ar thot)
28          phrase+(ot aud+audi ses+(ar spec:dejs:hall-json) ~)
[ hoons/arvo/mar/hall/action.hoon 30:9 Test::Whitespace tallColsig joined 0-running runstep #11 @29:9; comment overindented by 4 ]
29!         ::
30>         notify+(ot aud+audi pes+(mu pres) ~)
31          naming+(ot aud+audi man+huma ~)
[ hoons/arvo/mar/hall/action.hoon 33:9 Test::Whitespace tallColsig joined 0-running runstep #13 @32:9; comment overindented by 4 ]
32!         ::
33>         glyph+(ot gyf+so aud+audi bin+bo ~)
34          nick+(ot who+(su fed:ag) nic+so ~)
[ hoons/arvo/mar/hall/action.hoon 36:9 Test::Whitespace tallColsig joined 0-running runstep #15 @35:9; comment overindented by 4 ]
35!         ::
36>         public+(ot add+bo cir+circ ~)
```

**Comment by ohAitch** [comment]
This seems very similar to the question of jog-aligned vs rune-aligned
comments - I think there the conclusion was also "allow both", so sounds
good

**Comment by jeffreykegler** [comment]
Implemented for 0-runnings in commit f11cfc0 as referenced above.  I'll propagate it to the others as I come across the warnings.  With this I think I can mark this "ready for closing".

**Comment by jeffreykegler** [comment]
Closed as per the above.
